packages/openapi-generator/src/generator.ts

```diff
@@ -75,10 +74,23 @@ async function generateSDKSources(
   openApiDocument: OpenApiDocument,
   options: GeneratorOptions
 ): Promise<void> {
+  // TODO: This isn't really "request builder" anymore
   logger.info(`Generating request builder in ${serviceDir}.`);
   // TODO: what about overwrite?
+  if (openApiDocument.schemas.length) {
+    const modelDir = resolve(serviceDir, 'model');
+    await createInterfaceFiles(modelDir, openApiDocument);
```

Comment:
we have an inconsistent notion of `model`, `interfaces` and `schemas` which I think are roughly the same. From the old api generator the word `model` is familiar on typescript they are `interfaces` and in the open api spec they are called `schemas`. I think I would use a a folders `schemas` and use the word schema in the code. On the source code level they are of course `interfaces`.

Reply:
but this is up for discussio.

Reply:
After reading a bit I understand the distinction a bit better like model for parisng cause related to openApi lagnuage, schema for evalutaiton cause spec is evaluated and we write than interface. Nevertheless I found it hard to find the where this is done place.

Reply:
I like this, done.